website/docs/repositories.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Extra Repository Configs
+---
+
+When you need to reference packages from repositories not in the worker's
+repository config you can use the `extra_repos` field in the [dependencies
+section](spec#dependencies-section) section to inject extra repositories at
+the right time so those dependencies can be satisified.
+
+The `extra_repos` field takes a list of repository configs with the following
+structure:
+
+
+- **`keys`**
+  A map of keys required to enable the configured repositories. Each key in
+  this map is associated with a specific source and must be imported to allow
+  the repositories to function as expected. The content of this is a
+  [source](sources.md) just like in the sources section.
+
+- **`config`**
+  A collection of repository configurations to add to the environment. The
+  structure and format of these configurations are specific to the distribution
+  being used (e.g., apt or yum configurations). The content of this is a
+  [source](sources.md) just like in the sources section.
+
+- **`data`**
+  A list of additional data required to support the repository configurations.
+  This may include files or other resources that are essential for certain
+  configurations—especially file-backed repositories that need data not already
+  available in the environment. The content of this is a mount which takes
```

Comment:
Could you please reword this? I am not sure I understand the part of `certain configurations—especially`
Maybe something like:
```suggestion
This may include files or other resources essential for certain configurations,
particularly file-backed repositories that need data not already available in the
environment. The content of this is a mount which takes.
```